Code/Tools/ProjectManager/Source/ProjectExportWorker.cpp

```diff
@@ -213,7 +225,31 @@ namespace O3DE::ProjectManager
             return AZ::Failure(error);
         }
 
-        return AZ::Success();
+        // Fetch the output directory if the process was successful
+        // All standardized O3DE scripts end with a log line stating the output directory in quotes as the last word
+        // Use regex to collect that directory, and collect the slice of the word surrounded in quotes
+        QString exportOutputLastLine = exportOutput.split('\n', Qt::SkipEmptyParts).takeLast();
```

Comment:
No need to split and read the last line. If we make the project export line consistent and unique, the regex will find it. Doing this split is allocating additional strings (for each line) unnecessarily.